Makefile

```diff
@@ -1,15 +1,35 @@
-
 .PHONY: download-submodules
-download-submodules:
+download-submodules: check-git
 	git submodule init
 	git submodule update
 
+.PHONY: check-git
+check-git:
+	@which git > /dev/null || (echo "git is not installed. Please install and try again."; exit 1)
+
+.PHONY: check-go-tools
+check-go-tools:
+	@which go > /dev/null || (echo "Go is not installed.. Please install and try again."; exit 1)
+	@which go-bindata > /dev/null || (echo "go-bindata is not installed. Please install and try again."; exit 1)
```

Comment:
Unit tests and E2E tests are failing because this prerequisite is not met.

Btw, I think that it is legacy, as I'm unaware that `make bindata` is used (it certainly isn't for polybft consensus protocol), so it probably can be removed.

Reply:
Ok. Removed it then.